website/docs/reference/resource-configs/updated_at.md

```diff
@@ -27,6 +27,12 @@ snapshots:
 
 </File>
 
+:::caution
+
+You may get a warning if the data type of the `updated_at` column does not match the adapter-configured default.
```

Comment:
Suggest putting might instead of may. We generally favor it as it translates easier when we get to the point we'll support multiple languages
```suggestion
You might get a warning if the data type of the `updated_at` column does not match the adapter-configured default.
```